tools/releaser/scripts/breaking-changes.sh

```diff
@@ -3,15 +3,15 @@ set +e
 GOPATH=$(go env GOPATH)
 
 # Inputs:
-# GIT_BASE_REF - The base REF of the git repository
+# GIT_BASE_REF - The base REF of the git repository (git rev-parse origin/main)
 # Usually "${{ github.event.pull_request.base.sha || github.event.merge_group.base_sha }}" 
 # TARGET_BREAKING_CHANGES_FILE - file to save breaking changes
 script_path=$(dirname "$0")
 
 echo "Installing go-apidiff"
-go install github.com/joelanford/go-apidiff@latest
+go install github.com/joelanford/go-apidiff@latest > /dev/null
 
-GIT_BASE_REF=${GIT_BASE_REF:-"origin/main"}
+GIT_BASE_REF=${GIT_BASE_REF:-git rev-parse head || echo}
```

Comment:
This is much better for testing as we can then run workflow from separate branches with multiple corner cases without editing workflow